tests/cypress/integration/check-block-pattern-exists.test.js

```diff
@@ -0,0 +1,19 @@
+describe('Command: checkBlockPatternExists', () => {
+  before(() => {
+    cy.login();
+    cy.deactivatePlugin('classic-editor');
+  });
+
+  it('Should be able to Check Block Pattern Exists', () => {
+    cy.checkBlockPatternExists({
+      title: 'Quote',
+      categoryValue: 'text',
+    }).then(exists => {
+      if (exists) {
+        alert('The block patter exists!');
+      } else {
+        alert('The block pattern does not exist!');
+      }
```

Comment:
@faisal-alvi `alert()` requires human interaction to close the alert which never happens in the CI environments. I believe this causes the tests to be stuck.